src/components/Footer/Footer.js

```diff
@@ -1,7 +1,130 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import KakaoTalk from '../../assets/images/footer/kakao-talk.png';
+import './Footer.scss';
 
 const Footer = () => {
-  return <div>Footer</div>;
+  const loginToken = localStorage.getItem('signup_token');
+  const navigate = useNavigate();
+  const handleLink = id => {
+    if (id === 1) {
+      //로그인 or 로그아웃
+      if (!loginToken) navigate('/login');
+      else {
+        localStorage.removeItem('signup_token');
+        navigate('/');
+      }
+    } else {
+      navigate('/');
+    }
+  };
+  return (
+    <div className="footer">
+      <div className="container">
+        <div className="top">
+          <ul>
+            {loginToken ? (
+              <>
+                {FOOTER_TOP_ITEMS_LOGINED.map(footerTopItem => {
+                  return (
+                    <span
+                      key={footerTopItem.id}
+                      onClick={() => handleLink(footerTopItem.id)}
+                      className="footerTopItem"
+                    >
+                      <li>{footerTopItem.text}</li>
+                    </span>
+                  );
+                })}
+              </>
+            ) : (
+              <>
+                {FOOTER_TOP_ITEMS_NOT_LOGINED.map(footerTopItem => {
+                  return (
+                    <span
+                      key={footerTopItem.id}
+                      onClick={() => handleLink(footerTopItem.id)}
+                      className="footerTopItem"
+                    >
+                      <li>{footerTopItem.text}</li>
+                    </span>
+                  );
+                })}
+              </>
+            )}
```

Comment:
```suggestion
   <span onClick={handleLink} className="footerTopItem">
              <li>{loginToken ? '로그아웃' : '로그인'}</li>
            </span>
            {INFO_LIST.map(footerTopItem => {
              return (
                <span
                  key={footerTopItem.id}
                  onClick={() => navigate('/')}
                  className="footerTopItem"
                >
                  <li>{footerTopItem.text}</li>
                </span>
              );
            })}
```

로그인과 로그아웃만 변하고 나머지 부분들은 동일하기에
위와 같은 방식으로 변경해 주는 것이 더 효율적으로 느껴집니다!

Reply:
넵! 감사합니다. 말씀하신대로 반영하도록 하겠습니다